.env.development

```diff
@@ -3,6 +3,7 @@ FAST_REFRESH=true
 SKIP_PREFLIGHT_CHECK=true
 GENERATE_SOURCEMAP=false
 REACT_APP_FEATURE_CHARTS=true
+REACT_APP_SOCKET_API_KEY=72a5b4b0-e727-48be-8aa1-5da9d62fe635
```

Comment:
We don't need to commit the key to the repo. Isn't it gonna be handled by the CI even if it's not in the repo @berteotti? I mean, it will end up in both the build code and the requests in the end, but we can avoid having it here right?

Reply:
For the previews we need to add it on netlify. I think we can remove this and everyone who sets their env vars keys locally.

Reply:
Also for production

Reply:
For production we can still have them locally, but we need to be aware that there are missing keys when deploying.

Reply:
Fair enough, keys are leaked on requests anyway so they will never be a secret. We can leave them there.